File: ProblemSet/463. 整数排序/java/463 · 整数排序 选择排序.java
public class Solution {
    /**
     * @param A: an integer array
     * @return: nothing
     */
    public void sortIntegers(int[] A) {
        for (int i = 0; i < A.length; ++i) {
            int minIndex = i;
            for (int j = i + 1; j < A.length; ++j) 
                if (A[j] < A[minIndex]) 
                    minIndex = j;
            int tmp = A[i];
            A[i] = A[minIndex];
            A[minIndex] = tmp;
        }
    }
}

File: ProblemSet/463. 整数排序/python/463 · 整数排序 选择排序.py
class Solution:
    """
    @param A: an integer array
    @return: nothing
    """
    def sortIntegers(self, A):
        for i in range(len(A)):
            min_index = i
            for j in range(i + 1, len(A)):
                if A[j] < A[min_index]:
                    min_index = j
            A[i], A[min_index] = A[min_index], A[i]